users/config.rst

```diff
@@ -980,6 +1173,75 @@ setLowPriority
     disable this behavior, for example to control process priority yourself
     as part of launching Syncthing, set this option to ``false``.
 
+.. option:: options.maxFolderConcurrency
+
+    This option controls how many folders may concurrently be in I/O-intensive
+    operations such as syncing or scanning.  The mechanism is described in
```

Comment:
Just a side note, as I know this text comes from the already existing subpage, but in most cases syncing is not I/O intensive, especially outside of a local LAN, where bandwidth is almost always the limit, not the I/O. Even on a local LAN, I have personally not seen a case of I/O being the problem. It has always been the connection itself.

Only a digression; nothing to do here.